src/routes/chat/components/message/components/ImageAttachment.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import s from './ImageAttachment.css';
+
+function renderImage(attachments) {
+  return (
+    <div className={s.imageAttachments}>
+      <img alt="message attachments" src={attachments} />
```

Comment:
Pls call that prop in singular, and make it telling, like `attachmentSrc`

Reply:
I was thinking of a future option when the attachments will have an array that may contain more than one link. I also thought about the voiced messages that will be stored in the attachments.

Reply:
That's called 'overengineering' and that's bad, stick to what's required now, call it in singular